exdb/templates/exdb/pending.html

```diff
@@ -0,0 +1,46 @@
+{% extends "exdb/base.html" %}
+{% load staticfiles %}
+{% block css %}
+    {{ block.super }}
+    <link rel="stylesheet" type="text/css" href="{% static 'exdb/css/pending.css' %}"/>
+{% endblock %}
+{% block javascript %}
+    {{ block.super }}
+    <script type="text/javascript" src="{% static 'exdb/js/pending.js' %}"></script>
+{% endblock %}
+{% block body %}
```

Comment:
There's no need to override the `body` block if you're not changing anything in it. For this page, it seems that only the `content` block needs to be overridden.